data-model-changes.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#(set -o igncr) 2>/dev/null
+set -o igncr; # this comment is required to trick cygwin into dealing with windows vs. linux EOL characters
+
+function helpFunction {
+   echo ""
+   echo "Connection options:"
+   echo -e "\t-h database server host or socket directory"
+   echo -e "\t-u database user name"
+   exit 1 # Exit script after printing help
+}
+
+while getopts 'h:u:?' flag; do
+    case "${flag}" in
+        h) HOST="${OPTARG}" ;;
+        u) USER="${OPTARG}" ;;
+        *) helpFunction ;;
+    esac
+done
+
+# Print helpFunction in case parameters are empty
+if [ -z "$HOST" ] || [ -z "$USER" ]
```

Comment:
Can we default to `localhost` and `openchpl_dev` if they're empty? I think we do that in some of the scripts in the `dev/` directory, if you'd like to see a paradigm for doing that